gui.js

```diff
@@ -4472,22 +4511,6 @@ IDE_Morph.prototype.getURL = function (url) {
     }
 };
 
-IDE_Morph.prototype.getURLsbeOrRelative = function (url) {
```

Comment:
Yes, we want to load everything relative to ```snap.html```, that's right. However, I'm not sure what exactly 'loading' means, because XHR2 doesn't support the ```file:``` protocol (I think).